### DISCUSSION TITLE
--time option

### DISCUSSION THREAD

**Discussion by rakamakaf0n** [Serene-Arc/bulk-downloader-for-reddit#490] [Q&A]
Hi!
I have a problem with understanding how --time option works.
I'm using command 

`py -m bdfr download %download_path% --submitted --user %username% --sort new --time day --no-dupes --log %log_path%`

And as I understand this script, it must download submitted posts sorted by new for the last 24 hrs. But it starts to check every submitted post from first to last.
Explain pls how does it work or how should I use this command properly. Thx.

**Comment by Serene-Arc** [comment]
That is how to use it. It's worth noting that the time function works the same way it does on the Reddit and may not be exact.
